index.html

```diff
@@ -26,6 +26,223 @@ <h1>Hey There. I’m Alex</h1>
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>  
+
+                <div class="grid-item">
+                    <div class="multi-post2">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="button">
+                            <div class="flex-container">
+                                <div><button type="button">html</button></div>
+                                <div><button type="button">Bootstrap</button></div> 
+                                <div><button type="button">Ruby</button></div>  
+                              </div>
+                        </div>
+                        
+                    </div>
+                    <button class="big-button" type="button">See Project</button>
+                </div>
+
+                <div class="grid-item about">
+                    
+                        <h1>About me</h1>
+                   <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                   </p>
+                   <div class="design">
+                   <button class="big-button3" type="button">Get My Resume</button>
+                   </div>
+                </div>
+
+                <div class="grid-item languages">
+                     <h2><img class="imageTitle1" alt="languages" src="imgs/languages.svg"></h2>
+                  <ul>
+                         <li>Javascript</li>
+                         <li>Ruby</li>
+                         <li>Html</li>
+                         <li>Css</li>
+                     </ul>
+
+                     <h2><img class="imageTitle2" alt="languages" src="imgs/Frameworks.svg"></h2>
+
+                     <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                     <h2><img class="imageTitle3" alt="languages" src="imgs/skills.svg"></h2>
+
+                     <ul>
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                  
+              </div>
```

Comment:
- [ ] Kindly note that according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), you are supposed to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10). I have noticed that your Languages, Frameworks, and Shills are not centered, please kindly fix it :grin: :grin: 

Your design
![Screenshot from 2022-07-02 18-10-25](https://user-images.githubusercontent.com/74501523/177008266-f3c39a99-10c4-484d-b4d8-a75710f62ac6.png)

Figma design
![Screenshot from 2022-07-02 18-11-04](https://user-images.githubusercontent.com/74501523/177008271-ebffc133-e2d8-4837-87c4-d3bbcc6aa600.png)